From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Ayotte <martinayotte@yahoo.ca>
Date: Thu, 13 Dec 2018 17:09:54 -0500
Subject: [ARCHEOLOGY] add missing u-boot-meson64 patch for overlays

> X-Git-Archeology: - Revision c779ae2faadbfd5472927f58742cdb2371944935: https://github.com/armbian/build/commit/c779ae2faadbfd5472927f58742cdb2371944935
> X-Git-Archeology:   Date: Thu, 13 Dec 2018 17:09:54 -0500
> X-Git-Archeology:   From: Martin Ayotte <martinayotte@yahoo.ca>
> X-Git-Archeology:   Subject: add missing u-boot-meson64 patch for overlays
> X-Git-Archeology:
> X-Git-Archeology: - Revision e610f00bc7fdcab5ba4c648fa21148518b4b767b: https://github.com/armbian/build/commit/e610f00bc7fdcab5ba4c648fa21148518b4b767b
> X-Git-Archeology:   Date: Thu, 20 Jan 2022 23:18:55 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Bump Meson64 u-boot to 2022.01 (#3418)
> X-Git-Archeology:
> X-Git-Archeology: - Revision de26797423e22d58ec2882d7032c67f77196ecc5: https://github.com/armbian/build/commit/de26797423e22d58ec2882d7032c67f77196ecc5
> X-Git-Archeology:   Date: Sun, 06 Nov 2022 20:32:46 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move all legacy u-boot patches under one general legacy folder (#4386)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 24938adafeea9cd158557891eb85ea0730b41852: https://github.com/armbian/build/commit/24938adafeea9cd158557891eb85ea0730b41852
> X-Git-Archeology:   Date: Fri, 14 Jun 2024 00:34:34 +0200
> X-Git-Archeology:   From: Viacheslav Bocharov <adeep@lexina.in>
> X-Git-Archeology:   Subject: JetHub D2 (j200) add u-boot 2024.01 patches
> X-Git-Archeology:
> X-Git-Archeology: - Revision 654d208053f60d5793b25ea354df80a8d2116779: https://github.com/armbian/build/commit/654d208053f60d5793b25ea354df80a8d2116779
> X-Git-Archeology:   Date: Wed, 03 Jul 2024 11:26:07 +0200
> X-Git-Archeology:   From: Viacheslav Bocharov <adeep@lexina.in>
> X-Git-Archeology:   Subject: JetHome: Update JetHub family
> X-Git-Archeology:
---
 arch/arm/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 111111111111..222222222222 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -811,6 +811,8 @@ config ARCH_OMAP2PLUS
 config ARCH_MESON
 	bool "Amlogic Meson"
 	select GPIO_EXTRA_HEADER
+	select OF_LIBFDT
+	select OF_LIBFDT_OVERLAY
 	imply DISTRO_DEFAULTS
 	imply DM_RNG
 	help
-- 
Armbian

